crash-handler-process/platforms/upload-window-win.cpp

```diff
@@ -60,31 +60,40 @@ LRESULT CALLBACK FrameWndProc(HWND hwnd, UINT msg, WPARAM wParam, LPARAM lParam)
 	return DefWindowProc(hwnd, msg, wParam, lParam);
 }
 
-void UploadWindow::showButtons(bool ok_enabled, bool yes_enabled, bool cancel_enabled, bool no_enabled)
+void UploadWindow::showButtons(bool ok, bool yes, bool cancel, bool no)
 {
-	ShowWindow(ok_button_hwnd, ok_enabled ? SW_SHOW:SW_HIDE);
-	ShowWindow(yes_button_hwnd, yes_enabled ? SW_SHOW:SW_HIDE);
-	ShowWindow(cancel_button_hwnd, cancel_enabled ? SW_SHOW:SW_HIDE);
-	ShowWindow(no_button_hwnd, no_enabled ? SW_SHOW:SW_HIDE);
+	ShowWindow(ok_button_hwnd, ok ? SW_SHOW:SW_HIDE);
+	ShowWindow(yes_button_hwnd, yes ? SW_SHOW:SW_HIDE);
+	ShowWindow(cancel_button_hwnd, cancel ? SW_SHOW:SW_HIDE);
+	ShowWindow(no_button_hwnd, no ? SW_SHOW:SW_HIDE);
+}
+
+void UploadWindow::enableButtons(bool ok, bool yes, bool cancel, bool no)
+{
+	EnableWindow(ok_button_hwnd, ok );

```

Comment:
extra space before paranthesis. not sure if intended?

Reply:
fixed